src/Serialization/basic_types.jl

```diff
@@ -42,17 +42,24 @@ end
 @registerSerializationType(fmpq)
 
 function save_internal(s::SerializerState, q::fmpq)
-    return Dict(
-        :num => save_type_dispatch(s, numerator(q)),
-        :den => save_type_dispatch(s, denominator(q))
-    )
+    return string(q)
 end
 
-function load_internal(s::DeserializerState, ::Type{fmpq}, q::Dict)
-    return fmpq(load_type_dispatch(s, fmpz, q[:num]),
-                load_type_dispatch(s, fmpz, q[:den]))
+function load_internal(s::DeserializerState, ::Type{fmpq}, q::String)
+    fraction_parts = collect(map(String, split(q, "//")))
+    fraction_parts = [fmpz(s) for s in fraction_parts]
```

Comment:
I was wondering why we can't just do
```
    fraction_parts = map(fmpz, split(q, "//"))
```
which lead me to make https://github.com/Nemocas/Nemo.jl/pull/1375. So:
```suggestion
    # TODO: simplify the code below once https://github.com/Nemocas/Nemo.jl/pull/1375
    # is merged and in a Nemo release
    fraction_parts = collect(map(String, split(q, "//")))
    fraction_parts = [fmpz(s) for s in fraction_parts]
```